src/content/reference/react/use-server.md

```diff
@@ -203,4 +201,15 @@ function LikeButton() {
 }
 ```
 
+```js
+// actions.js
```

Comment:
I'm fleshing out the action as I want to illustrate that you need the directive on the module level